src/app/search/items/search-filters/loadouts.ts

```diff
@@ -47,7 +46,7 @@ const loadoutFilters: ItemFilterDefinition[] = [
             loadout.name.toLowerCase().includes(filterValue) ||
             (filterValue.startsWith('#') && // short circuit for less load
               !isInGameLoadout(loadout) &&
-              getHashtagsFromNote(loadout.notes)
+              getHashtagsFromString(loadout.name, loadout.notes)
```

Comment:
This might not actually have been searching the loadout name as advertised?

Reply:
I think it was, in the `loadout.name.toLowerCase().includes(filterValue)` line. It just wasn't doing any fancy hashtag normalization.

Reply:
Ah, completely true thank you.